src/main/scala/esmeta/lang/util/Parser.scala

```diff
@@ -1202,6 +1202,7 @@ trait Parsers extends IndentParsers {
 
   // rarely used expressions
   lazy val specialTy: P[Ty] = opt("an " | "a ") ~> {
+    "sequence of Unicode code points" ^^^ StrT |
```

Comment:
Is the sequence of Unicode code points is String type?

Reply:
This is in line with current [tymodel](https://github.com/es-meta/esmeta/blob/9acc5430167456f2c98f473c720340974dfcb5a4/src/main/scala/esmeta/ty/TyModel.scala#L358), which treats EcmascriptFunctionObject.[[SourceText]] as a StrT(https://tc39.es/ecma262/2022/#table-internal-slots-of-ecmascript-function-objects).